scanner/decisions/0009-nvd-cvss-scores.md

```diff
@@ -0,0 +1,145 @@
+Note: NVD tracks CVEs, so all of its data is CVE-based.
+
+Currently, Scanner V4 provides a single CVSS score and severity, which is preferably from the vendor.
+In reality, the CVSS scores that are shown are from NVD except in the following cases:
+
+* RHEL-base images
+  * Red Hat-provided CVSS scores/severities are used.
+* OSV-provided data
+  * If OSV.dev has a CVSS score, then that score is used.
+    * The severity is derived from the score based on the CVSS version's specification.
+  * Note: From personal experience, it is very common for the score displayed by OSV is the same as NVD.
+
+Currently, there is only a need to have CVSS scores from the vendor (if available) and NVD (if available), no more.
+However, it is possible this requirement changes in the future, and we may want to support more than just two
+sources for vulnerability data.
+
+The focus is on CVEs and RHSAs, RHEAs, and RHBAs. This document will not consider other types of advisories such as
+Ubuntu's USNs.
+
+The current API looks like the following:
+
+```
+message VulnerabilityReport {
+  message Vulnerability {
+    enum Severity {
+      SEVERITY_UNSPECIFIED = 0;
+      SEVERITY_LOW = 1;
+      SEVERITY_MODERATE = 2;
+      SEVERITY_IMPORTANT = 3;
+      SEVERITY_CRITICAL = 4;
+    }
+    message CVSS {
+      message V2 {
+        float base_score = 1;
+        string vector = 2;
+      }
+      message V3 {
+        float base_score = 1;
+        string vector = 2;
+      }
+      V2 v2 = 1;
+      V3 v3 = 2;
+    }
+    ...
+    string severity = 6;
+    ...
+    CVSS cvss = 12;
+  }
+  ...
+}
+```
+
+## Decision
+
+The VulnerabilityReport API will be extended as follows:
+
+```
+message VulnerabilityReport {
+  message Vulnerability {
+    enum Severity {
+      SEVERITY_UNSPECIFIED = 0;
+      SEVERITY_LOW = 1;
+      SEVERITY_MODERATE = 2;
+      SEVERITY_IMPORTANT = 3;
+      SEVERITY_CRITICAL = 4;
+    }
+    message CVSS {
+      message V2 {
+        float base_score = 1;
+        string vector = 2;
+      }
+      message V3 {
+        float base_score = 1;
+        string vector = 2;
+      }
+      V2 v2 = 1;
+      V3 v3 = 2;
+      string updater = 3; <-- New field.
+      string cvss_url = 4; <-- New field
+    }
+    ...
+    string severity = 6;
+    ...
+    CVSS cvss = 12;
+    repeated CVSS cvss_metrics = 13; <-- New field.
+  }
+  ...
+}
+```
+
+There will be a new type plus two new fields added:
+
+* `CVSS.updater`
+  * This specifies the source of the particular CVSS metrics. The value will be the name of the scanner updater, indicating the data source from which the updater is fetching vulnerabilities.
+* `CVSS.cvss_url`
+  * This specifies the CVSS score source URL, aiding both API and UI users in tracking the origin of this metric.
+* `cvss_metrics`
+  * This is a list of each unique CVSS metric based on the source.
```

Comment:
Can you provide an example?

Reply:
Will this include what is in the original `cvss` field?

Reply:
@RTann 
Yes
**_cvss_metrics_** will also include contents in original **_cvss_** field. 
> This approach simplifies data querying and filtering, as `cvss_metrics` will be the sole field used for filtering data or making policies.